repos/system_upgrade/common/actors/scandnfpluginpath/libraries/scandnfpluginpath.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from leapp.libraries.stdlib import api
+from leapp.models import DnfPluginPathDetected
+
+DNF_CONFIG_PATH = '/etc/dnf/dnf.conf'
+
+
+def _read_dnf_config(config_path):
+    if os.path.isfile(config_path):
+        with open(config_path, 'r') as config:
+            content = config.read()
+            return content
+    return ''
+
+
+def _is_pluginpath_set(config_content):
+    for raw_line in config_content.splitlines():
+        line = raw_line.strip()
+        if not line or line.startswith('#'):
+            continue
+        if line.lower().startswith('pluginpath'):
+            return True
+    return False
+
+
+def scan_dnf_pluginpath():
+    """Scan DNF configuration and produce DnfPluginPathDetected message."""
+    content = _read_dnf_config(DNF_CONFIG_PATH)
+    if content:
+        is_detected = _is_pluginpath_set(content)
+    else:
+        is_detected = False
```

Comment:
nitpick; can be skipped / kept as it is

```suggestion
    is_detected = False 
    if content: 
        is_detected = _is_pluginpath_set(content)
```